stages/9-openebs-install-check/openebs-data-plane-test

```diff
@@ -17,6 +17,15 @@ kubectl get po -n openebs
 git clone https://$username:$password@github.com/mayadata-io/oep.git
 cd oep
 
+####################################
+##  Sequencing and Running test   ##
+####################################
+
+bash utils/pooling jobname:openebs-control-plane-test
+bash utils/e2e-cr jobname:openebs-data-plane-test jobphase:Running
+bash utils/e2e-cr jobname:openebs-install-check jobphase:Waiting
```

Comment:
in the job script openebs-control-plane-test we have already set openebs-install-check jobphase:Waiting then why are you we doing it again in this script